support-frontend/assets/pages/[countryGroupId]/student/components/StudentLandingPageGlobal.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import { type LandingPageVariant } from 'helpers/globalsAndSwitches/landingPageSettings';
+import type {
+	ActiveProductKey,
+	ActiveRatePlanKey,
+} from 'helpers/productCatalog';
+import type { GeoId } from 'pages/geoIdConfig';
+import { StudentLandingPageGlobal } from './StudentLandingPageGlobal';
+
+const oneYearStudentDiscount = {
+	amount: 9,
+	periodNoun: 'year',
+	discountPriceWithCurrency: '£9',
+	fullPriceWithCurrency: '£120',
+};
+
+describe('<StudentLandingPageGlobal />', () => {
+	const geoId: GeoId = 'us';
+	const productKey: ActiveProductKey = 'SupporterPlus';
+	const ratePlanKey: ActiveRatePlanKey = 'Monthly';
+	const landingPageVariant = {
+		products: {
+			SupporterPlus: {
+				benefits: ['Benefit 1', 'Benefit 2'],
+			},
+		},
+	} as unknown as LandingPageVariant;
```

Comment:
Does the `as unknown` means other  `LandingPageVariant` required fields are ignored (as not used in this test)?  Have not seen that before.

Reply:
Test passes fine